sections/production/installpackageswithnpmci.md

```diff
@@ -0,0 +1,30 @@
+# Install packages with npm ci in production
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+You locked your dependencies following [**Lock dependencies**](/sections/production/lockdependencies.md) but you now need to make sure those exact package versions are used in production.
+
+Using `npm ci` to install packages will do exactly that and more.
+* It will only install packages in the `dependencies` section of your `package.json`. `devDependencies` are ignored
+* It will fail if your `package.json` and your `package-lock.json` do not match (they should) or if you don't have a lock file
+* If a `node_modules` folder is present it will automatically remove it before installing
+* It is faster! Nearly twice as fast according to the release blog post (see link below)
```

Comment:
Could be nice to also link it here

Reply:
So you would have the link in both sections? Or just here?